turbo/jsonrpc/trace_filtering.go

```diff
@@ -66,14 +66,13 @@ func (api *TraceAPIImpl) Transaction(ctx context.Context, txHash common.Hash, ga
 		return nil, err
 	}
 
+	txNumsReader := rawdbv3.TxNums.WithCustomReadTxNumFunc(freezeblocks.ReadTxNumFuncFromBlockReader(ctx, api._blockReader))
```

Comment:
This move of `txNumsReader` doesn't agree with its move in PR #14470

Reply:
We would need to port this change as well: 44443ebba80a9518c8e264e7be9dbcbecb1ee409, should I do that?

Reply:
Ah, I see